src/Shared/CertificateGeneration/UnixCertificateManager.cs

```diff
@@ -15,7 +23,19 @@ internal UnixCertificateManager(string subject, int version)
         {
         }
 
-        public override bool IsTrusted(X509Certificate2 certificate) => false;
+        public override bool IsTrusted(X509Certificate2 certificate)
+        {
+            // Return true when all stores trust the cert.
```

Comment:
This could be extended to indicate 'partial': some locations trust the certificate, while others do not.
Or it can return `true` when at least one trust the certificate.